Update Readme to match the template (#2200)
ansible-collections/amazon.aws: README.md

<!--end collection content-->

## Installation

> Before using this collection, you need to install it with the Ansible Galaxy command-line tool:

Reviewer: ```suggestion
The amazon.aws collection can be installed with Ansible Galaxy command-line tool:
```

    version: 3.1.1
```

> or using the ansibble-galaxy command as follows

Reviewer: ```suggestion
A specific version of amazon.aws collection can be installed using the ansible-galaxy command as follows
```

Author: The line - "A specific version of amazon.aws collection can be installed" , is already mentioned previously. I will fix typo.

> or using the ansibble-galaxy command as follows

Reviewer 2: ```suggestion
or using the ansible-galaxy command as follows
```

or use @mandar242's suggestion

See [CONTRIBUTING.md](https://github.com/ansible-collections/amazon.aws/blob/main/CONTRIBUTING.md) for more details.

### More information about contributing

- [Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html) - Details on contributing to Ansible
- [Contributing to Collections](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#contributing-to-collections) - How to check out collection git repositories correctly
- [Guidelines for Ansible Amazon AWS module development](https://docs.ansible.com/ansible/latest/collections/amazon/aws/docsite/dev_guidelines.html)
- [Getting Started With AWS Ansible Module Development and Community Contribution](https://www.ansible.com/blog/getting-started-with-aws-ansible-module-development)

## Support

You can also join us on:

- Libera.Chat IRC - the ``#ansible-aws`` [irc.libera.chat](https://libera.chat/) channel

## Release notes

See the [rendered changelog](https://ansible-collections.github.io/amazon.aws/branch/main/collections/amazon/aws/docsite/CHANGELOG.html) or the [raw generated changelog](https://github.com/ansible-collections/amazon.aws/tree/main/CHANGELOG.rst).

> ## Related Information

Reviewer: maybe something like this?
```suggestion
## Useful Resources
```

Author: I have used what is given in the template. I think that is the requirement.
